frontend/src/components/form/api/ApiWrapper.vue

```diff
@@ -5,7 +5,7 @@ Wrapper component for form components to save data back to API
 <template>
   <ValidationObserver ref="validationObserver" v-slot="validationObserver" slim>
     <v-form
-      :class="[{ 'api-wrapper--inline': !autoSave && !readonly && !separateButtons }]"
```

Comment:
why this change?

Reply:
Ui, good that you caught that.
Was to Test that the Styles still apply